robe.el

```diff
@@ -135,8 +145,9 @@ project."
                          (cond
                           ((string-match "robe on \\([0-9]+\\)" s)
                            (setq started t)
-                           (setq robe-port (string-to-number
-                                            (match-string 1 s))))
+                           (when (not robe-port)
```

Comment:
`robe-port` is never cleared. What happens if we stop one Robe session (by killing the REPL) and then try to start another? All that without `docker-robe`, of course.